src/main/java/Lesson7/UpperCase.java

```diff
@@ -0,0 +1,10 @@
+package Lesson7;
+
+public class UpperCase {
+
+    public static String toUpperCase(String textInput){
+        if (textInput == null || textInput.length() == 0) throw new IllegalArgumentException("textInput can not be null or empty String");
+        else{
+            return textInput.toUpperCase();
+    }
+}}
```

Comment:
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html
CTRL+ALT+L 

aby trochę posprzątać formatowanie tego tekstu ;)